packages/strapi-plugin-content-type-builder/admin/src/components/HeaderNavLink/Wrapper.js

```diff
@@ -11,6 +11,8 @@ const Wrapper = styled.div`
         z-index: 99;
       `;
     }
+
+    return '';
```

Comment:
You don't need to return this if you set this CSS attribute like the comment above
```
${({ isActive }) => isActive && css`
  height: 3rem;
  color: #007eff;
  font-weight: 600;
  border-bottom: 2px solid #007eff;
  z-index: 99;
`}

Reply:
Well it's best if we can  avoid using `css`

> A helper function to generate CSS from a template literal with interpolations. You need to use this if you return a template literal with functions inside an interpolation due to how tagged template literals work in JavaScript.
If you're interpolating a string you do not need to use this, only if you're interpolating a function.

So not using it will improve performances a bit.